docs/source/en/tasks/object_detection.mdx

```diff
@@ -0,0 +1,487 @@
+...         image = np.array(image.convert("RGB"))[:, :, ::-1]
+...         out = transform(image=image, bboxes=objects["bbox"], category=objects["category"])
+
+...         area.append(objects["area"])
+...         images.append(torch.tensor(out["image"]).flip(-1).permute(2, 0, 1))
+...         bboxes.append(out["bboxes"])
+...         categories.append(out["category"])
+
+...     targets = [
+...         {"image_id": id_, "annotations": formatted_anns(id_, cat_, ar_, box_)}
+...         for id_, cat_, ar_, box_ in zip(image_ids, categories, area, bboxes)
+...     ]
+
+...     return image_processor(images=images, annotations=targets, return_tensors="pt")
+```
+
+Apply this preprocessing function to the entire dataset using 🤗 Datasets `with_transform` method. This method applies
+transformations on the fly when you load an element of the dataset.
+
+At this point, you can check what an example from the dataset looks like after the transformations. You should see a tensor
+with `pixel_values`, a tensor with `pixel_mask`, and `labels`.
+
+```py
+>>> cppe5["train"] = cppe5["train"].with_transform(transform_aug_ann)
+>>> cppe5["train"][15]
+```
+
+You have successfully augmented the individual images and prepared their annotations. However, preprocessing isn't
+complete yet. In the final step, create a custom `collate_fn` to batch images together.
+Pad images (which are now `pixel_values`) to the largest image in a batch, and create a corresponding `pixel_mask`
+to indicate which pixels are real (1) and which are padding (0).
+
+```py
+>>> def collate_fn(batch):
+...     pixel_values = [item["pixel_values"] for item in batch]
+...     encoding = image_processor.pad_and_create_pixel_mask(pixel_values, return_tensors="pt")
+...     labels = [item["labels"] for item in batch]
+...     batch = {}
+...     batch["pixel_values"] = encoding["pixel_values"]
+...     batch["pixel_mask"] = encoding["pixel_mask"]
+...     batch["labels"] = labels
+...     return batch
+```
+
+## Training the DERT model
+You have done most of the heavy lifting in the previous sections, so now you are ready to train your model!
+The images in this dataset are still quite large, even after resizing. This means that finetuning this model will
+require at least one GPU.
+
+```py
+>>> import torch
+
+>>> device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+```
+
+Training involves the following steps:
+1. Load the model with `DetrForObjectDetection.from_pretrained` using the same checkpoint as in the preprocessing.
+2. Define your training hyperparameters in `TrainingArguments`.
+3. Pass the training arguments to `Trainer` along with the model, dataset, image processor, and data collator.
+4. Call `train()`` to finetune your model.
+
+When loading the model from the same checkpoint that you used for the preprocessing, remember to pass the label to
+id and id to label maps that you created earlier from the dataset's metadata.
```

Comment:
I think it's better to refer those maps as `label2id` and `id2label`, respectively.

Reply:
Fixed in 1a38260b27ef977e0d82ba1c918eeb19c8f3dcb7